### ISSUE TITLE
How to do multiprocessing in dataloader?

### ISSUE THREAD

**Issue by NNPanNPU** [funcwj/conv-tasnet#3]
This code really helps. However, when the batch size is large, loading data becomes a bottleneck for training. There's no implementation of multi worker in dataloader. Hope to solve this problem. Any suggestions about this issue?

**Comment by funcwj** [comment]
I have done that and I will update data loader soon.

**Comment by funcwj** [comment]
I fixed it on the branch 'paralle-loader'. You can try it and if errors existed, please report them here.

**Comment by NNPanNPU** [comment]
Thanks for updating and getting back to me. I tried the code and fixed some bugs, which are

1. line 18 in dataset.py.
The original one  `dataset = Dataset(shuffle=train, **data_kwargs)`
should be `dataset = Dataset(**data_kwargs)`, since Dataset does not have input "shuffle".

2. line 39--49 in train.py
The original one
`train_loader = make_dataloader(train=True,
                                   data_kwargs=train_data,
                                   batch_size=args.batch_size,
                                   chunk_size=args.chunk_size,
                                   num_workers=args.num_workers)
    dev_loader = make_dataloader(train=False,
                                 data_kwargs=dev_data,
                                 batch_size=args.batch_size,
                                 chunk_size=args.chunk_size,
                                 num_workers=args.num_workers)`

where `chunk_size=args.chunk_size`,  should be `chunk_size=chunk_size`.

**Comment by funcwj** [comment]
Thanks, I will fix them soon
